yt/python/yt/environment/yt_env.py

```diff
@@ -66,6 +67,11 @@
 _environment_driver_logging_config = None
 
 
+DEFAULT_ADMIN_USERNAME = "admin"
+DEFAULT_ADMIN_PASSWORD = "admin"
+DEFAULT_ADMIN_TOKEN = "admin"
```

Comment:
k8s operator uses "admin" "password" "password"

https://github.com/ytsaurus/ytsaurus-k8s-operator/blob/main/pkg/consts/defaults.go

https://github.com/ytsaurus/ytsaurus-k8s-operator/blob/7d512528654291ce841e5f468cd01becde77bc12/pkg/components/master.go#L120

Reply:
okay, let it be "password"